_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -6334,6 +6337,34 @@ mfxStatus MfxHwH264Encode::CheckRunTimeExtBuffers(
             checkSts = sts;
     }
 
+    for (mfxU16 fieldId = 0; fieldId < NumFields; fieldId++)
+    {
+        mfxExtAVCRoundingOffset* extRoundingOffset =
+                reinterpret_cast<mfxExtAVCRoundingOffset*>(MfxHwH264Encode::GetExtBuffer(ctrl->ExtParam, ctrl->NumExtParam, MFX_EXTBUFF_AVC_ROUNDING_OFFSET, fieldId));
+        if (extRoundingOffset)
+        {
+            if (extRoundingOffset->EnableRoundingIntra == MFX_CODINGOPTION_ON)
+            {
+                if (extRoundingOffset->RoundingOffsetIntra < 0 ||
```

Comment:
As I see from API, this field is declared unsigned

Reply:
yes, it's unsigned, just want to show the range explicitly here. will remove it.